build.sbt

```diff
@@ -20,7 +9,16 @@ lazy val `scala-parser-combinators` = crossProject.in(file(".")).
   settings(
     moduleName         := "scala-parser-combinators",
     version            := "1.1.0-SNAPSHOT",
-    scalaVersion       := crossScalaVersions.value.head
+    scalaVersion       := crossScalaVersions.value.head,
+    crossScalaVersions := {
```

Comment:
Hum, you can't do that. `crossScalaVersions` is *supposed* to be in `ThisBuild`. In subprojects, `crossScalaVersions` is just plain ignored, so effectively that's the same as not writing the setting at all. Hence the Travis failures.